.travis/deploy.sh

```diff
@@ -9,7 +9,19 @@ ssh-add .travis/id_rsa
 git remote add deploy ssh://$DEPLOY_USER@$IP/~/$DEPLOY_DIR
 git push deploy master
 
-# SSH into the remote server and start NEXT Directory Platform
+# If the commit message contains the phrase 'RESET_NEXT_STORAGE', this deployment will 
```

Comment:
Is this information (reset to genesis via commit pattern) documented anywhere else than here (the Wiki or the README)?

Reply:
This information is currently only documented here. I can add this information to the 'Deploy to Non-Localhost ' section in the README if this is the preferred location. Or should we have a Wiki section for this information?